src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X500NameEncoder.ManagedDecode.cs

```diff
@@ -135,7 +135,9 @@ internal static partial class X500NameEncoder
                     decodedName.Append(dnSeparator);
                 }
 
-                return decodedName.ToString();
+                string name = decodedName.ToString();
+                decodedName.Dispose();
```

Comment:
This was a little odd to me, but is consistent with what I observed for other uses of `ValueStringBuilder`. It can't be in a using because it's a `ref struct` that's passed by reference to another method.

All the `Dispose` does is return rented buffers, if any, to the array pool, so it isn't catastrophic if we fail to dispose.